ads/google/a4a/traffic-experiments.js

```diff
@@ -62,19 +62,21 @@ const INTERNALLY_SELECTED_ID = '2088462';
  * @param {!ExperimentInfo} internalBranches experiment and control branch IDs to
  *   use when experiment is triggered internally (i.e., via client-side
  *   selection).
+ * @param {!string} controlWithDelayedId  Experiment ID string for the branch
```

Comment:
As I think about this more: I've done what you describe in your first sentence, and am testing now. I agree with what you say in the rest of your comment, but push is happening in 100 minutes and it's not critical for this. So my preference would be to push what I've got, and work on the refactor as a separate issue. Okay?